components/dash-table/src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -177,7 +177,11 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
                     active.row !== active_cell?.row)
             ) {
                 const target = this.$el.querySelector(
-                    `td[data-dash-row="${active_cell.row}"][data-dash-column="${active_cell.column_id}"]:not(.phantom-cell)`
+                    `td[data-dash-row="${
+                        active_cell.row
+                    }"][data-dash-column="${CSS.escape(
+                        active_cell.column_id
+                    )}"]:not(.phantom-cell)`
```

Comment:
That became a bit harder to read on multiple line, I would destructure the `active_cell` and put the escaped column id in a variable.

Reply:
dunno if it's net easier to read or not, but I DRYed this in https://github.com/plotly/dash/pull/2619/commits/3391ee9910359115489b4763a59c94128a1a6089